### PR TITLE
Swagger documentation for endpoints

### PR DESCRIPTION
**Pull request by Gabriele-ele**

### CODE REVIEW

File: src/WebAPI/Controllers/ItemController.cs [hunk 1]
```diff
@@ -35,13 +56,24 @@ public async Task<IActionResult> Create(ItemCreate itemDto)
         return Ok(new { response });
     }
 
+
```
Comment (ghost) [inline src/WebAPI/Controllers/ItemController.cs]: remove 1 empty line

File: src/WebAPI/Program.cs [hunk 2]
```diff
@@ -4,13 +4,19 @@
 using Microsoft.EntityFrameworkCore;
 using Microsoft.OpenApi.Models;
 using Serilog;
+using System.Reflection;
 using System.Text.Json.Serialization;
 using WebAPI.Middleware;
 
 namespace WebAPI;
-
+/// <summary>
```
Comment (ghost) [inline src/WebAPI/Program.cs:11]: after namespace add empty line

**Review by arturascervinskas — APPROVED** [review]

File: src/WebAPI/Program.cs [hunk 3]
```diff
@@ -40,12 +45,24 @@ public static void Main(string[] args)
 
         builder.Logging.AddSerilog(logger);
 
+        builder.Services.AddHttpClient();
```
Comment (ghost) [inline src/WebAPI/Program.cs]: Why Do you need this service

**Review by ViktorasSce — APPROVED** [review]

File: src/WebAPI/Program.cs [hunk 4]
```diff
@@ -40,12 +45,24 @@ public static void Main(string[] args)
...
+        builder.Services.AddSwaggerGen(c =>
+        {
+            c.SwaggerDoc("v1", new OpenApiInfo { Title = "CacheTask", Version = "v1" });
+
+            // Include the XML comments file
+            var xmlFile = $"{Assembly.GetExecutingAssembly().GetName().Name}.xml";
+            var xmlPath = Path.Combine(AppContext.BaseDirectory, xmlFile);
+            c.IncludeXmlComments(xmlPath);
+        });
+
```
Comment (ghost) [inline src/WebAPI/Program.cs]: this breaks authentication, update swager definition that is in JWTAuthenticateInjection.cs

**Review by ghost — APPROVED** [review]

**Pull request merged**